docs/index.html

```diff
@@ -33,7 +33,7 @@
 								</a>
 							</li>
 							<li>
-								<a href="https://publicslack.com/slacks/gardener-cloud/invites/new" target="_blank">
+								<a href="https://kubernetes.slack.com/messages/gardener" target="_blank">
```

Comment:
This direct URL requires that the user already has access to the kubernetes slack workspace.
Should we instead (or additionally) link the kubernetes slack invite link http://slack.k8s.io/ ?

Reply:
I think this link is appropriate to directly jump into our channel.
We can document maybe with a tip on the link to go to http://slack.k8s.io/ for an invitation.

Reply:
/lgtm